backend/src/main/kotlin/com/manage/crm/config/web/compression/gzip/GzipCompressionUtils.kt

```
@@ -0,0 +1,61 @@
+        }
+
+        private fun compressBytes(bytes: ByteArray): ByteArray {
+            try {
+                ByteArrayOutputStream().use { byteArrayOutputStream ->
+                    GZIPOutputStream(byteArrayOutputStream).use { gzipOutputStream ->
+                        gzipOutputStream.write(bytes)
+                        gzipOutputStream.finish()
+                    }
+                    return byteArrayOutputStream.toByteArray()
+                }
+            } catch (e: IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)